.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -4,8 +4,11 @@ about: Suggest an idea for this project
 
 ---
 
+**Describe the problem (if applicable)**
```

Comment:
This implies that every feature we implement is based on a problem or defect, but we should stay positive :-)

I'm not sure what needs improvement here. There is nothing that prevents the author from describing the current state, but normally it's enough to describe the desired change. Maybe it's enough to rename "Technical details" to "Solution ideas / technical details"?